"""
Подвиг 1. Имеется следующий фрагмент программы:

try:
    val = float(input())
except ValueError as e:
    print(e)
else:
    val *= 10
    print(val)

При каких входных данных выполнение программы перейдет в блок else?

всегда будет переходить, вне зависимости от возникновения исключений
    при вводе любых вещественных значений
при вводе любых не числовых значений
    при вводе любых целочисленных значений
"""
